modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -0,0 +1,112 @@
+
+Runtime Fabric uses its own registry to store the necessary Docker images for installing and managing Runtime Fabric and for running applications in Kubernetes. If your security policies require that you pull your Docker images from a registry under your control, you can set up a local registry to pull and store these Docker images.
+
+== Limitations for Local Registry Support 
+
+Review the following limitations about support for local registries: 
+
+* Runtime Fabric supports local registries with Runtime Fabric on Self-Managed Kubernetes only.
+
+* You can use a local registry only if you create a new Runtime Fabric using version 1.12.0 or higher. You cannot add a local registry to an existing Runtime Fabric installation (regardless of its version) even if you upgrade that installation to 1.12.0 or higher. 
+
+* You must use xref:install-rtfctl.adoc[Runtime Fabric command line utility (`rtfctl`)] version 0.3.150 or higher.
+
+* If you create a Runtime Fabric to use with a local registry, you cannot later reconfigure it to use the `rtf-runtime-registry` endpoint. To use the `rtf-runtime-registry` endpoint, you must install a new Runtime Fabric using the standard installation procedure.
+
+* You're responsible for synchronization between your registry and the Runtime Fabric registry. MuleSoft doesn't provide additional software to synchronize images between the registries.
+
+* If you use a local registry that needs authentication, synchronization and propagation of a pull-secret between different namespaces is your responsibility. Alternatively, Runtime Fabric can synchronize your pull-secret across different namespaces if your secret uses the following label:
+
+----
+  labels:
+    rtf.mulesoft.com/synchronized: "true"
+----
+
+* If you need to authenticate access to your local registry, ensure the https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#registry-secret-existing-credentials[corresponding credentials are configured as a Kubernetes secret in the `rtf-` namespace^]. You'll need the name of the configured secret when you install Runtime Fabric. 
+
+* You also need to synchronize in your registry the Mule runtime versions you intend to use for application deployments. Refer to the xref:release-notes::runtime-fabric/runtime-fabric-runtimes-release-notes.adoc[Runtime Fabric Patch Update Release Notes] for information about the latest Mule runtime patch versions. 
+
+== Configure a Local Registry for Use with Runtime Fabric 
+
+To configure a local registry you must gather and add the necessary credentials to synchronize the registries.
+
+=== Before You Begin
+
+Ensure you've performed the following tasks: 
+
+. Setup, configure, and test a local Docker image registry.
+. Synchronize to your local registry all the Docker images you need to install Runtime Fabric:
++
+[NOTE]
+Dependency versions are specific to the Runtime Fabric version.
```

Comment:
I suggest moving this so that it does not interrupt the flow between the list introduction and the list items.